### DISCUSSION TITLE
what happens if the workflows to run contains an actions/cache action?

### DISCUSSION THREAD

**Discussion by ozeta** [nektos/act#1527] [Q&A] [answered]
as stated by the title. Does act uses the real cache , if it exists on gh, or ignore it and skip this step?

**Comment by KnisterPeter** [comment, marked as answer]
If you do not provide a cache implementation it does fail.
Have a look here for more information: https://github.com/nektos/act/issues/1513

**Reply by ozeta** [reply]
Do you have any insight about download/upload-artifact action?

**Reply by KnisterPeter** [reply]
The artifact actions should work if you start act with the embedded artifact server.
Please have a look at our docs: https://github.com/nektos/act#flags

**Reply by ozeta** [reply]
much appreciated! :D 👍
